ibm/service/appconfiguration/data_source_ibm_app_config_segments.go

```diff
@@ -0,0 +1,473 @@
+							Description: "Segment id.",
+						},
+						"description": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "Segment description.",
+						},
+						"tags": {
+							Type:        schema.TypeString,
+							Optional:    true,
+							Description: "Tags associated with the segments.",
+						},
+						"created_time": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Creation time of the segment.",
+						},
+						"updated_time": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Last modified time of the segment data.",
+						},
+						"href": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Segment URL..",
+						},
+						"features": {
+							Type:        schema.TypeList,
+							Computed:    true,
+							Description: "List of Features associated with the segment.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"feature_id": {
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "feature id.",
+									},
+									"name": {
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "Name of the Feature.",
+									},
+								},
+							},
+						},
+						"properties": {
+							Type:        schema.TypeList,
+							Computed:    true,
+							Description: "List of properties associated with the segment.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"property_id": {
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "property id.",
+									},
+									"name": {
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "Name of the Property.",
+									},
+								},
+							},
+						},
+						"rules": {
+							Type:        schema.TypeList,
+							Computed:    true,
+							Description: "List of rules that determine if the entity belongs to the segment during feature / property evaluation.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"values": {
+										Type:        schema.TypeList,
+										Computed:    true,
+										Description: "List of values. Entities matching any of the given values will be considered to belong to the segment",
+										Elem: &schema.Schema{
+											Type: schema.TypeString,
+										},
+									},
+									"attribute_name": {
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "Attribute name.",
+									},
+									"operator": {
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "Operator to be used for the evaluation if the entity belongs to the segment.",
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+			"next": {
```

Comment:
since pagination is already handled..
pagination related attributes like next, first, previous, last are not required

Reply:
Addressed